Artur_Badalyan/python/25May2021/perfectNum.py

```diff
@@ -0,0 +1,21 @@
+
+def perfectNum (num):
+    sum=0
+    for i in range(1,num):
+        if num % i == 0:
+            sum = sum + i
+    
+    if num == sum:
+        print(num,"is a perfect number")
+    else:
+        print(num,"is not a perfect number")
+
+
+print("Write number for check is a perfect number or not.")
+num=input()
+
+if num.isdigit():
+     perfectNum(int(num))
+else:
+     print("User input is string or small then 0")
+
```

Comment:
1. perfectNum -> coding style
2. limitation for big number
![image](https://user-images.githubusercontent.com/80816144/119567866-e1d17000-bdbd-11eb-9cbe-7845da5a4474.png)